packages/twenty-server/src/engine/workspace-manager/workspace-sync-metadata/standard-agents/agents/workflow-creation-agent.ts

```diff
@@ -0,0 +1,33 @@
+import { type StandardAgentDefinition } from 'src/engine/workspace-manager/workspace-sync-metadata/standard-agents/types/standard-agent-definition.interface';
+
+export const WORKFLOW_CREATION_AGENT: StandardAgentDefinition = {
```

Comment:
This kind of business logic does not belong in the engine but instead belongs in the business modules (workflow?). The agent should only contain the abstractions